src/scripts/show_originals.css

```
@@ -1,7 +1,33 @@
-.xkit-show-originals-hidden article {
+[data-show-originals="on"] .xkit-show-originals-hidden article {
   display: none;
 }
 
 .xkit-show-originals-lengthened {
   min-height: 100vh;
 }
+
+.xkit-show-originals-controls {
+  color: var(--blog-title-color, rgb(var(--white-on-dark)));
+  display: flex;
+  font-size: 1.125rem;
+  font-weight: 700;
+  line-height: 1.333;
+  margin-bottom: 20px;
+}
+.xkit-show-originals-controls > a {
+  padding: 14px 16px;
+  text-decoration: none;
+  text-transform: capitalize;
+}
+.xkit-show-originals-controls > a:not([data-mode="disabled"]) {
+  cursor: pointer;
+}
+.xkit-show-originals-controls > a:hover {
+  background: var(--blog-background-hover-color, rgba(var(--white-on-dark), .13));
+}
+[data-show-originals="on"] .xkit-show-originals-controls > a[data-mode="on"],
+[data-show-originals="off"] .xkit-show-originals-controls > a[data-mode="off"],
+.xkit-show-originals-controls > a[data-mode="disabled"] {
+  box-shadow: inset 0 -2px 0 var(--blog-link-color, rgb(var(--accent)));
```

Makes the enabled shadow thicker. No solid justification for this one either, it just looks better to me.

```suggestion
  box-shadow: inset 0 -3px 0 var(--blog-link-color, rgb(var(--accent)));
```